apisix/plugins/openid-connect.lua

```diff
@@ -130,6 +130,28 @@ local schema = {
                 "header to the request for downstream.",
             type = "boolean",
             default = false
+        },
+        proxy_opts = {
+            description = "access via openid server via a proxy server ",
```

Comment:
Typo

Reply:
> Typo?

seems to, i just fixed it
by the way, do you think it will be better by using "oauth2 server" instead of "openid server" ?

Reply:
Do you mean `OpenID Provider(OP)`?

Reply:
This PR will be handed over to you to follow up later, I have to fix other problems. Can you? @darkSheep404